src/locksManager.js

```diff
@@ -54,4 +54,20 @@ var LocksManager = GObject.registerClass({
         const variant = new GLib.Variant('a{sb}', {locked: false});
         this._proxy.SetSync(lock, variant);
     }
+
+    getTrapSequence(lock) {
+        return this._proxy.getDictValueSync(lock, 'trap_sequence');
+    }
+
+    setLockTried(lock, tried) {
+        try {
+            const [variant] = this._proxy.GetSync(lock);
+            const dict = new GLib.VariantDict(variant);
+            dict.insert_value('tried', new GLib.Variant('b', tried));
+            this._proxy.SetSync(lock, dict.end());
+        } catch (error) {
+            const variant2 = new GLib.Variant('a{sb}', {tried: true});
```

Comment:
This needs to be `{tried}` as otherwise the `tried` parameter given to the function is ignored, I think?

Also, a pre-existing problem, but I would really like to understand and fix where this discrepancy in variant types with the game state server is coming from, it caused problems with the hackdex corruption. I'll open an issue for that.